api/all/src/main/java/io/opentelemetry/api/trace/TraceFlags.java

```diff
@@ -45,17 +36,15 @@ static TraceFlags getSampled() {
   }
 
   /**
-   * Returns the {@link TraceFlags} converted from the given lowercase hex (base16) representation.
+   * Returns the {@link TraceFlags} converted from the given two lowercase hex characters.
    *
-   * @param src the buffer where the hex (base16) representation of the {@link TraceFlags} is.
-   * @param srcOffset the offset int buffer.
+   * @param first the first hex character.
+   * @param second the second hex character.
    * @return the {@link TraceFlags} converted from the given lowercase hex (base16) representation.
-   * @throws NullPointerException if {@code src} is null.
-   * @throws IndexOutOfBoundsException if {@code src} is too short.
-   * @throws IllegalArgumentException if invalid characters in the {@code src}.
+   * @throws IllegalArgumentException if invalid lowercase hex characters.
    */
-  static TraceFlags fromHex(CharSequence src, int srcOffset) {
-    return ImmutableTraceFlags.fromHex(src, srcOffset);
+  static TraceFlags fromHex(char first, char second) {
```

Comment:
Alternative names "fromHexCharacters" or "fromHexChars" or this is good enough :) ?

Reply:
I'd very strongly prefer not to make any more breaking API changes at this point. Let's leave it as-is.